core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/ExposedService.java

```diff
@@ -15,9 +15,9 @@
     ALL("Every Service", "ALL", "", "", true, null, null, false, true),
 
     CLOUDERA_MANAGER("CM-API", "CM-API", "CM-API", "/cm-api/", true, 7180, 7183, true, true),
-    CLOUDERA_MANAGER_UI("CM-UI", "CM-UI", "CM-UI", "/cmf/home", true, 7180, 7183, false, false),
+    CLOUDERA_MANAGER_UI("CM-UI", "CM-UI", "CM-UI", "/cmf/home/", true, 7180, 7183, false, false),
     HUE("HUE", "HUE_LOAD_BALANCER", "HUE", "/hue/", true, 8889, 8889, false, false),
-    WEBHDFS("WebHDFS", "NAMENODE", "WEBHDFS", "/webhdfs/v1", true, 9870, 9871, true, true),
+    WEBHDFS("WebHDFS", "NAMENODE", "WEBHDFS", "/webhdfs/v1/", true, 9870, 9871, true, true),
```

Comment:
if we are making a change to WEBHDFS: might want it to be `/webhdfs/v1/?OP=LISTSTATUS` which makes it a valid url in the UI. Otherwise if you click on it there is a failure.

I don't know what the long term way to display API endpoints, but sharing if trying to make a valid endpoint. There are other API calls in here that aren't valid if you click on them. So this recommendation can totally be ignored just wanted to share :)

Reply:
I could add that, but I guess the main purpose for the API endpoint is for programatic consumption and might add extra overhead to people if they have to remove the part from the URL.